domains/client/domain-operator/src/domain_worker_starter.rs

```diff
@@ -152,6 +152,7 @@ pub(super) async fn start_worker<
     let handle_slot_notifications_fut = handle_slot_notifications::<Block, CBlock, _, _>(
         consensus_client.as_ref(),
         move |consensus_block_info, slot_info| {
+            info!("🔖 Producing bundle at slot {:?}", slot_info.slot);
```

Comment:
This log is not accurate, we just received a new slot notification here, the node may not produce bundle if:
- it can't solve the challenge
- it hasn't finished producing ER for the last domain block
- there is no extrinsic in the tx pool as this check was just added in this commit.